### ISSUE TITLE
investigate missing a11y snapshots

### ISSUE THREAD

**Issue by gbinal** [GSA/site-scanning#1906]
https://api.gsa.gov/technology/site-scanning/data/archive/json/weekly-snapshot-accessibility-details-2025-04-24.json works and I think there was supposed to be a fresh file every day after that, but at some point since then, it appears that has broken.  

note the following not working: 
- https://api.gsa.gov/technology/site-scanning/data/archive/json/weekly-snapshot-accessibility-details-2025-12-17.json
- https://api.gsa.gov/technology/site-scanning/data/archive/json/weekly-snapshot-accessibility-details-2026-04-17.json
- etc.

**Comment by ethangardner** [comment]
@gbinal - this will be fixed the next time we deploy. I left detailed notes in https://github.com/GSA/site-scanning-engine/pull/536, but the TLDR is that the accessibility files were being saved with the full timestamp, `weekly-snapshot-accessibility-details-2026-05-24T12:58:22.593Z.json` instead of the `YYYY-MM-DD` format.

**Comment by gbinal** [comment]
strange - not sure what's up with that but let's ship for now and then I'll make an issue for these other items.
